internal/servers/console_proxy_core_test.go

```
@@ -0,0 +1,354 @@
+	err      error
+}
+
+func (m *mockPrivateConsoleServer) ResolveTarget(
+	ctx context.Context,
+	req *privatev1.ConsoleResolveTargetRequest,
+) (*privatev1.ConsoleResolveTargetResponse, error) {
+	return m.response, m.err
+}
+
+// pipeRWC wraps a net.Conn as io.ReadWriteCloser for testing.
+type pipeRWC struct {
```

golangci-lint (2.12.2)

[error] type pipeRWC is unused

(unused)